server/app/views/applicant/NorthStarAddressCorrectionBlockView.java

```diff
@@ -20,100 +20,98 @@
+                params.messages());
+        context.setVariable("pageTitle", pageTitle);
 
-    context.setVariable("programTitle", params.programTitle());
-    context.setVariable("programDescription", params.programDescription());
-    context.setVariable("confirmAddressAction", getFormAction(params, applicantRequestedAction));
-    context.setVariable("goBackAction", goBackAction(params));
-    context.setVariable("addressSuggestionGroup", addressSuggestionGroup);
-    context.setVariable("isEligibilityEnabled", isEligibilityEnabled);
-    context.setVariable("applicationParams", params);
+        context.setVariable("programTitle", params.programTitle());
+        context.setVariable("programDescription", params.programDescription());
+        context.setVariable("confirmAddressAction", getFormAction(params, applicantRequestedAction));
+        context.setVariable("goBackAction", goBackAction(params));
+        context.setVariable("addressSuggestionGroup", addressSuggestionGroup);
+        context.setVariable("isEligibilityEnabled", isEligibilityEnabled);
+        context.setVariable("applicationParams", params);
 
-    // Progress bar
-    ProgressBar progressBar =
-        new ProgressBar(params.blockList(), params.blockIndex(), params.messages());
-    context.setVariable("progressBar", progressBar);
+        // Progress bar
+        ProgressBar progressBar = new ProgressBar(params.blockList(), params.blockIndex(), params.messages());
+        context.setVariable("progressBar", progressBar);
 
-    boolean anySuggestions = addressSuggestionGroup.getAddressSuggestions().size() > 0;
-    context.setVariable("anySuggestions", anySuggestions);
+        boolean anySuggestions = addressSuggestionGroup.getAddressSuggestions().size() > 0;
+        context.setVariable("anySuggestions", anySuggestions);
 
-    String alertMessage =
-        anySuggestions
-            ? params.messages().at(MessageKey.ADDRESS_CORRECTION_FOUND_SIMILAR_LINE_2.getKeyName())
-            : params.messages().at(MessageKey.ADDRESS_CORRECTION_NO_VALID_LINE_2.getKeyName());
+        String alertMessage = anySuggestions
+                ? params.messages().at(MessageKey.ADDRESS_CORRECTION_FOUND_SIMILAR_LINE_2.getKeyName())
+                : params.messages().at(MessageKey.ADDRESS_CORRECTION_NO_VALID_LINE_2.getKeyName());
 
-    AlertSettings addressAlertSettings =
-        new AlertSettings(
-            /* show= */ true,
-            Optional.of(params.messages().at(MessageKey.ADDRESS_CORRECTION_LINE_1.getKeyName())),
-            alertMessage,
-            AlertType.WARNING);
-    context.setVariable("addressAlertSettings", addressAlertSettings);
+        AlertSettings addressAlertSettings = AlertSettings.builder()
+                .show(true)
+                .title(Optional.of(params.messages().at(MessageKey.ADDRESS_CORRECTION_LINE_1.getKeyName())))
+                .text(alertMessage)
+                .unescapedDescription(false)
```

Comment:
suggestion (blocking): Again, double check this, but I'm pretty sure `unescaptedDescription` defaulted to `true` previously.